tool/planet/etcd.go

```diff
@@ -198,7 +197,7 @@ func etcdUpgrade(rollback bool) error {
 	}
 
 	if env.Get(EnvEtcdProxy) == EtcdProxyOn {
-		log.Info("etcd is in proxy mode, nothing to do")
+		log.Info("etcd is in proxy mode, restarting etcd clients")
```

Comment:
Why is the message about restarting clients here and not in the method that does the restart?

Reply:
my bad, I originally added it the restart here, but then changed my mind and linked it into the service start code.